### ISSUE TITLE
Interface adjusting bug

### ISSUE THREAD

**Issue by kerbacho** [opentoonz/opentoonz#5310]
### Description

Open Toonz crashes as soon as I try to change the position of windows in a room

### Steps to Reproduce

1. undock a window in any room
2. dock it somewhere

### Expected Behavior

It will crash

### Screenshots, Video & Crash Logs

OpenToonz 1.7.1  (Build May 10 2023)

Report Date: 2024-02-24 13:39:39
Crash Reason: (SIGSEGV) Segmentation Fault

Build ABI: x86_64-little_endian-lp64
Operating System: macOS 10.16
OS Kernel: 23.3.0
CPU Threads: 16

GPU Vendor: ATI Technologies Inc.
GPU Model: AMD Radeon Pro 5500M OpenGL Engine
GPU Version: 2.1 ATI-5.2.5

Crash File: /Users/kerbacho/Library/Application Support/OpenToonz/OpenToonz/crash/Crash-20240224-133939.log

Application Dir: /Applications/OpenToonz.app/Contents/MacOS
Stuff Dir: /Applications/OpenToonz/OpenToonz_stuff

Project Name: sandbox
Scene Name: test
Project Path: /Applications/OpenToonz/OpenToonz_stuff/sandbox/sandbox_otprj.xml
Scene Path: /Applications/OpenToonz/OpenToonz_stuff/sandbox/scenes/test.tnz

==== Backtrace ====
  1> 3   libobjc.A.dylib                     0x00007ff80f9d5dff objc_destructInstance + 163
  2> 4   libqcocoa.dylib                     0x000000010416d114 qt_plugin_instance + 172644
  3>   4> sh: addr2line: command not found
  5> 7   AppKit                              0x00007ff81367ef5d _NSTSMEventHandler + 299
  6> 8   HIToolbox                           0x00007ff81a746703 _ZL23DispatchEventToHandlersP14EventTargetRecP14OpaqueEventRefP14HandlerCallRec + 1364
  7> 9   HIToolbox                           0x00007ff81a745b4f _ZL30SendEventToEventTargetInternalP14OpaqueEventRefP20OpaqueEventTargetRefP14HandlerCallRec + 336
  8> 10  HIToolbox                           0x00007ff81a7459ee SendEventToEventTargetWithOptions + 45
  9> 11  HIToolbox                           0x00007ff81a79eda1 SendTSMEvent_WithCompletionHandler + 388
 10> 12  HIToolbox                           0x00007ff81a8f74c0 __SendTextInputEvent_WithCompletionHandler_block_invoke + 511
 11> 13  HIToolbox                           0x00007ff81a8f5e08 SendTextInputEvent_WithCompletionHandler + 1046
 12> sh: addr2line: command not found
 13> sh: addr2line: command not found
 14> sh: addr2line: command not found
 15> 17  CoreFoundation                      0x00007ff80fe7bc3e __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
 16> 18  CoreFoundation                      0x00007ff80fe7bb73 __CFRunLoopDoBlocks + 398
 17> 19  CoreFoundation                      0x00007ff80fe7a9f6 __CFRunLoopRun + 940
 18> 20  CoreFoundation                      0x00007ff80fe7a082 CFRunLoopRunSpecific + 557
 19> 21  HIToolbox                           0x00007ff81a76c9d9 RunCurrentEventLoopInMode + 292
 20> 22  HIToolbox                           0x00007ff81a76c616 ReceiveNextEventCommon + 201
 21> 23  HIToolbox                           0x00007ff81a76c531 _BlockUntilNextEventMatchingListInModeWithFilter + 66
 22> 24  AppKit                              0x00007ff81348ac39 _DPSNextEvent + 880
 23>  24> sh: addr2line: command not found
 25> 27  libqcocoa.dylib                     0x000000010417286c qt_plugin_instance + 195004
 26> 28  QtCore                              0x000000010926b97f _ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 431
 27> 29  QtCore                              0x000000010926fd82 _ZN16QCoreApplication4execEv + 130
 28> 30  OpenToonz                           0x00000001034a7e07 main + 16487
 29> 31  dyld                                0x00007ff80fa15386 start + 1942
==== End ====

### OpenToonz Version

1.7

### Operating System

macOS

### GPU

AMD

### Graphics Tablet

Huion

**Comment by RodneyBaker** [comment]
I know in the past some users have reported issues with custom rooms but if these are the default rooms I believe this is the first I've heard of this.  More information will be appreciated

Make sure you are using the latest nightly when testing as that includes all bug fixes since the last annual release.

**Comment by kerbacho** [comment]
fixed it by deleting the cache!

**Comment by kerbacho** [comment]
Nope, it happens again! Deleting the cache doesn't fix it this time.

**Comment by RodneyBaker** [comment]
I can't replicate this on a Mac using the listed instructions.
I am operating Mac OS 10.15.7 though so... not sure if that makes a difference.

**Comment by andeon** [comment]
@RodneyBaker Apparently, this is caused by corrupted settings. The user needs to reset the profiles to recreate fresh settings, which may resolve the issue.

Anyway, it’s recommended to download the [nightly build](https://github.com/opentoonz/opentoonz/releases/tag/nightly), as it includes many important bug fixes. If the user still experiences problems after installing the new version, they can submit another issue report. Feel free to close this issue.
